configs/SPEEDYBEEF405V5/config.h

```diff
@@ -0,0 +1,131 @@
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F405
+
+#define BOARD_NAME        SPEEDYBEEF405V5
+#define MANUFACTURER_ID   SPBE
+
+#define USE_ACC
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+#define USE_BARO
+#define USE_BARO_DPS310
+#define USE_GYRO_CLKIN
+
+#define MOTOR1_PIN          PB1
+#define MOTOR2_PIN          PB0
+#define MOTOR3_PIN          PB10
+#define MOTOR4_PIN          PB11
+#define SERVO1_PIN          PC9
+#define SERVO2_PIN          PC8
+
+#define UART1_TX_PIN        PA9
+#define UART1_RX_PIN        PA10
+#define UART2_TX_PIN        PA2
+#define UART2_RX_PIN        PA3
+#define UART3_TX_PIN        PC10
+#define UART3_RX_PIN        PC11
+#define UART4_TX_PIN        PA0
+#define UART4_RX_PIN        PA1
+#define UART5_RX_PIN        PD2
+#define UART6_TX_PIN        PC6
+#define UART6_RX_PIN        PC7
+
+#define I2C1_SCL_PIN        PB6
+#define I2C1_SDA_PIN        PB7
+
+#define SPI1_SCK_PIN        PA5
+#define SPI1_SDI_PIN        PA6
+#define SPI1_SDO_PIN        PA7
+#define GYRO_1_EXTI_PIN     PC4
+#define GYRO_1_CS_PIN       PA4
+#define GYRO_1_CLKIN_PIN    PB15
+
+#define SPI2_SCK_PIN        PB13
+#define SPI2_SDI_PIN        PC2
+#define SPI2_SDO_PIN        PC3
+#define MAX7456_SPI_CS_PIN  PB12
+
+#define SPI3_SCK_PIN        PB3
+#define SPI3_SDI_PIN        PB4
+#define SPI3_SDO_PIN        PB5
+#define FLASH_CS_PIN        PC12
+
+#define LED0_PIN            PA15
+#define BEEPER_PIN          PB9
+#define RX_PPM_PIN          PA3
+#define LED_STRIP_PIN       PA8
+#define CAMERA_CONTROL_PIN  PB8
+#define ADC_VBAT_PIN        PC1
+#define ADC_RSSI_PIN        PC0
+#define ADC_CURR_PIN        PC5
+#define PINIO1_PIN          PC13
+#define PINIO2_PIN          PC14
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP(0,  MOTOR1_PIN,         2, 0)  \
+    TIMER_PIN_MAP(1,  MOTOR2_PIN,         2, 0)  \
+    TIMER_PIN_MAP(2,  MOTOR3_PIN,         1, 0)  \
+    TIMER_PIN_MAP(3,  MOTOR4_PIN,         1, 1)  \
+    TIMER_PIN_MAP(4,  SERVO1_PIN,         2, -1) \
+    TIMER_PIN_MAP(5,  SERVO2_PIN,         2, -1) \
+    TIMER_PIN_MAP(6,  LED_STRIP_PIN,      1, 1)  \
+    TIMER_PIN_MAP(7,  BEEPER_PIN,         2, -1) \
+    TIMER_PIN_MAP(8,  RX_PPM_PIN,         2, -1) \
+    TIMER_PIN_MAP(9,  CAMERA_CONTROL_PIN, 1, -1) \
+    TIMER_PIN_MAP(10, GYRO_1_CLKIN_PIN,   3, -1)
+
+#define ADC1_DMA_OPT   1
+#define SYSTEM_HSE_MHZ 8
+
+#define BEEPER_INVERTED
+#define MAG_I2C_INSTANCE      I2CDEV_1
+#define BARO_I2C_INSTANCE     I2CDEV_1
+#define GYRO_1_ALIGN          CW270_DEG
+#define GYRO_1_SPI_INSTANCE   SPI1
+#define MAX7456_SPI_INSTANCE  SPI2
+#define FLASH_SPI_INSTANCE    SPI3
+
+#define PINIO1_CONFIG 1
+#define PINIO2_CONFIG 1
+#define PINIO1_BOX 40
+#define PINIO2_BOX 41
+#define BOX_USER1_NAME "9V BEC OFF"
```

Comment:
Great idea to add these definitions. We should do this to all existing targets too.